### ISSUE TITLE
Support syncing multiple major versions

### ISSUE THREAD

**Issue by megawac** [jsdelivr/libgrabber#50]
Several projects maintain multiple major version (notably jQuery). It'd be useful to support syncing latest tags on different major versions

**Comment by lewisgoddard** [comment]
How would this be implemented? Something in the info.json? The only solution that I can think of that doesn't require manually enabling would be to pull in every release.

**Comment by megawac** [comment]
Nah, it shouldn't be too hard to determine when comparing `local` (jsdelivr) and `remote` versions; all that would have to be done would be to divy up versions by major and detect local major versions which aren't hosted. If any, open a PR.

Probably would make sense to only open 1 PR per project at a time.

**Comment by lewisgoddard** [comment]
What about systems that support, for example, 1.8.x and 1.9.x? This seems like it's going to fix jQuery (important, admittedly) but not much else.
